aspnetcore/security/authentication/sociallogins.md

```diff
@@ -15,135 +15,82 @@ uid: security/authentication/sociallogins
+* In Visual Studio, create a New Project (from the Start Page, or via **File > New > Project**):
 
 ![image](sociallogins/_static/new-project.png)
 
-* Tap **Web Application** and verify **Authentication** is set to **Individual User Accounts**
+* Tap **Web Application** and verify **Authentication** is set to **Individual User Accounts**:
 
 ![image](sociallogins/_static/select-project.png)
 
-* Enable SSL
+## Enable SSL
+
+Some external authentication providers will reject requests coming from origins that don't use the **https** protocol. This reflects the trend of major providers like [Google](https://security.googleblog.com/2014/08/https-as-ranking-signal_6.html) moving their public API services to https and discontinuing the use of unencrypted endpoints. We encourage you to follow this trend and enable SSL for your entire site.
 
-     * In solution explorer, right click the project and select **Properties**
+* In solution explorer, right click the project and select **Properties**.
 
-     * On the left pane, tap **Debug**
+* On the left pane, tap **Debug**.
 
-     * Check **Enable SSL**
+* Check **Enable SSL**.
 
-     * Copy the SSL URL and paste it into the **App URL**
+* Copy the SSL URL and paste it into the **App URL**:
 
 ![image](sociallogins/_static/ssl.png)
 
-* Require SSL. Modify the services.AddMvc(); code in `Startup` under `ConfigureServices`:
+* Modify the `services.AddMvc();` code in `Startup.cs` under `ConfigureServices` to reject all requests that are not coming over *https*:
 
 ````csharp
 services.AddMvc(options =>
 {
+    options.SslPort = 44321;
     options.Filters.Add(new RequireHttpsAttribute ());
 });
 ````
 
-* Test the app
-
-## Creating the app in Facebook
-
-Each of the OAuth2 providers require provider specific keys to enable OAuth2.
-
-* Navigate to [https://developers.facebook.com/apps](https://developers.facebook.com/apps) and log in.
-
-* If you aren’t already registered as a Facebook developer, click  *Register as a Developer* and follow the directions to register.
-
-* Tap **Add a New App**
-
-* Select **Website** from the platform choices.
-
-* Tap **Skip and Create App ID**
-
-![image](sociallogins/_static/FBApp03.png)
-
-* Enter a display name,  category, contact email and tap **Create App ID**.
-
-![image](sociallogins/_static/FBApp04.png)
-
-* Tap **Settings** from the left menu bar.
-
-![image](sociallogins/_static/FBApp05.png)
-
-* On the **Basic** settings section of the page select Add Platform to specify that you are adding a website app.
-
-![image](sociallogins/_static/FBApp06.png)
-
-* Select Website from the platform choices.
+* Test the app to ensure that static files are still being served and publicly exposed routes are accessible.
+   * There shouldn't be any warnings logged to the browser console in Developer Tools.
+   * Attempting to navigate to the previous URI that used the *http* protocol should now result in **connection rejected** errors from the browser or a blank page.
```

Comment:
Making the change here. It's impossible to standardize on this everywhere due to varying terminology from various login providers. Microsoft Account provider, for example, even uses URL and URI on the same page for its UI.